e2e_tests/integration/play-command.spec.ts

```diff
@@ -18,11 +18,10 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-/* global Cypress, cy, test, expect, before */
+/* global Cypress, cy, before */
 
+import { ClearEditorButton } from './0.index.spec'
```

Comment:
Looks a bit weird importing these from a .spec file. You mentioned you were looking into best practises, did you find a better way?

Reply:
The recommended way I found was to just put the selectors in the spec file they're used, as we're doing, but since typescript thinks they are in the same scope I thought either import or declare in several places and silence typescript with `ts-expect-error`. But come to think of it, it's cleaner to remove them and add a custom clear command instead.

Reply:
Apparently importing anything from another spec causes all tests in the file with the export to be included in the suite that is importing, so they run several times, so that's a no-go. Each selector was only used once in each of the three suites so I decided to inline the selector.